src/actions/endpointExplorer.js

```diff
@@ -17,11 +17,11 @@ export function changePendingRequestProps(props) {
 }
 
 export const UPDATE_VALUES = "UPDATE_VALUES";
-export function updateValues(param, values) {
```

Comment:
If this action is becoming single-valued, you should reflect it in the name as well:  UPDATE_VALUE